### PR TITLE
KCRO-149: Implementation of Websites API

### PR DESCRIPTION
**Pull request by Navoda626**
KCRO-149: Implementation of Websites API

Implementation of websites API to facilitate adding websites using the comm.land. So an admin and users would be able to add custom websites with a logo.

A new document with the type 'website' is created in the database for every website added. 'name' and the 'web_url' are mandatory parameters for a website document and there is 'users' parameter which holds the list of users whom the website is assigned. And there is a 'account_wide' parameter works as a flag to assign the website to every user in that account.

Create, update, delete and read APIs implemented for the websites.

Additionally a binary logo can be updated to a website document and fetched from a website document.

### CODE REVIEW

File: kazoo_stdlib/include/kz_databases.hrl [hunk 1]
```diff
@@ -51,6 +52,7 @@
                        ,?KZ_ACDC_DB
                        ,?KZ_OAUTH_DB
                        ,?KZ_FUNCTIONS_DB
+                       ,?KZ_WEBSITES_DB
```
Comment (jamesaimonetti) [inline kazoo_stdlib/include/kz_databases.hrl]: Why this change? I don't see the need for a `websites` db nor do I see it used in the Crossbar PR?

File: kazoo_data/src/kzs_util.erl [hunk 2]
```diff
@@ -92,6 +92,7 @@ db_classification(?KZ_FAXES_DB) -> 'aggregate';
 db_classification(?KZ_ACDC_DB) -> 'aggregate';
 db_classification(?KZ_SERVICES_DB) -> 'aggregate';
 db_classification(?KZ_PORT_REQUESTS_DB) -> 'aggregate';
+db_classification(?KZ_WEBSITES_DB) -> 'aggregate';
```
Comment (jamesaimonetti) [inline kazoo_data/src/kzs_util.erl]: Seems unnecessary

**Review by jamesaimonetti — CHANGES_REQUESTED** [review]

File: kazoo_stdlib/include/kz_databases.hrl (hunk 1 above)
Comment (Navoda626) [reply kazoo_stdlib/include/kz_databases.hrl]: Yes this was initially added. This is not necessary. Will resolve this

File: kazoo_data/src/kzs_util.erl (hunk 2 above)
Comment (Navoda626) [reply kazoo_data/src/kzs_util.erl]: Yes this is not necessary. Will resolve this
